pkg/restore/restore.go

```diff
@@ -605,6 +560,104 @@ func (ctx *restoreContext) execute() (Result, Result) {
 	return warnings, errs
 }
 
+func (ctx *restoreContext) processSelectedResource(
```

Comment:
Adding a comment for this function would be helpful to understand what it's doing.

Reply:
Adding brief comment before the call, and longer comment at the func definition.